app/commands/delete_publish_intent.rb

```diff
@@ -2,7 +2,14 @@ module Commands
   class DeletePublishIntent < BaseCommand
```

Comment:
Can you add something in the commit message about why we're doing this please

Reply:
I've added a much more detailed explanation in the commit message.